### DISCUSSION TITLE
Can you share how to support plugins?

### DISCUSSION THREAD

**Discussion by ThaddeusJiang** [adithya-badidey/tiddlyjam#9] [Q&A]
Hi @adithya-badidey 

Can you share how to support plugins?

Once you share the How-To,
I'd like to learn and help support other plugins.

I am using 

- [ ] Google Analytics plugin
- [ ] Twitter Plugin

**Comment by adithya-badidey** [comment]
Hi Thaddeus, 

Could you provide a link to those plugins too? That would be easier.
